htdocs/projet/element.php

```diff
@@ -391,7 +391,7 @@
 		'buttonnew'=>'AddWarehouse',
 		'project_field'=>'fk_project',
 		'testnew'=>$user->hasRight('stock', 'creer'),
-		'test'=>isModEnabled('stock') && $user->hasRight('stock', 'lire') && getDolGlobalString('WAREHOUSE_ASK_WAREHOUSE_DURING_PROJECT')
+		'test'=>isModEnabled('stock') && $user->hasRight('stock', 'lire') // && getDolGlobalString('WAREHOUSE_ASK_WAREHOUSE_DURING_PROJECT')
```

Comment:
When WAREHOUSE_ASK_WAREHOUSE_DURING_PROJECT is not set, there is no project information linked to a warehouse. So no way to have a report of list of warehouse inside a project. So old test seems good to me. We don't show a page/link if this page is not relevant...

Reply:
Except that it is not possible to change this variable, which is only used in two places in dolibarr (the stock admin, the list of elements)
the main problem is that if we do not remove this variable, it is impossible to have the list of warehouses associated with the project...

![image](https://github.com/Dolibarr/dolibarr/assets/1179011/6b478c4c-18fe-4962-9d4d-4bbd1ffe9c3c)